internal/infrastructure/kubernetes/resource/resource_test.go

```diff
@@ -51,6 +52,19 @@ func TestExpectedServiceSpec(t *testing.T) {
 				ExternalTrafficPolicy: corev1.ServiceExternalTrafficPolicyTypeLocal,
 			},
 		},
+		{
+			name: "LoadBalancerWithAllocateLoadBalancerNodePorts",
+			args: args{service: &egcfgv1a1.KubernetesServiceSpec{
+				Type:                          egcfgv1a1.GetKubernetesServiceType(egcfgv1a1.ServiceTypeLoadBalancer),
+				AllocateLoadBalancerNodePorts: &allocateLoadBalancerNodePorts,
```

Comment:
can we use `ptrTo(true)` style in this file ?

Reply:
Sounds great. Updated.